variables.tf

```
@@ -1,3 +1,9 @@
+variable "api_gw_enabled" {
+  default     = false
+  description = "Install api gateway in commercial"
+  type        = bool
+}
```

Add `nullable = false` to avoid case requiring testing for null...

```suggestion
variable "api_gw_enabled" {
  default     = false
  description = "Install api gateway in commercial"
  nullable    = false
  type        = bool
}
```